### PR TITLE
Implement Kaplan's feedback re pipelines

### PR DESCRIPTION
**Pull request by Chr1st1anSears**
This PR makes a lot of changes, but the only page you really need to pay attention to is the one that's now called "Component deployment lifecycle." Otherwise, this PR adds and deletes some images, and it hides three pages about pipelines that Adam Kaplan said need to be rewritten.

Please take a look at that "Component deployment lifecycle" page and tell me what you think. Thanks!

### CODE REVIEW

File: docs/modules/ROOT/pages/getting-started/component_deployment_lifecycle.adoc [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+CP Development Team <cp-devel@redhat.com>
+:toc: left
+:icons: font
+:numbered:
+:source-highlighter: highlightjs
+
+== Scope
+Learn how to track the lifecycle of a component's deployment in the {ProductName} web console.
+
+== Web console visualization
```
Comment (dphilips-RH) [inline docs/modules/ROOT/pages/getting-started/component_deployment_lifecycle.adoc:11]: I might move this section to the end of the doc. I think both 'build pipeline customization' and 'logs' might get lost at the bottom underneath the table. 
On the other hand, this makes the most sense as the first article in this guide.

File: docs/modules/ROOT/pages/getting-started/component_deployment_lifecycle.adoc [hunk 2]
```diff
@@ -0,0 +1,50 @@
+= Component deployment lifecycle
```
Comment (dphilips-RH) [inline docs/modules/ROOT/pages/getting-started/component_deployment_lifecycle.adoc:1]: Could we link to a CLI option in this guide once we have the getting started CLI guide? I.E. how to monitor your components via CLI?

File: docs/modules/ROOT/pages/getting-started/component_deployment_lifecycle.adoc [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+|Environment (Static or Managed)
+|image:environment_icon.png[alt=Environment]
+
+|GitHub
+|image:github_icon.svg[alt=GitHub]
+
+|===
+
+== Build pipeline customization
```
Comment (dphilips-RH) [inline docs/modules/ROOT/pages/getting-started/component_deployment_lifecycle.adoc:46]: Not necessarily today, but we could add a screenshot here in the future? Though maintaining screenshots can be difficult

File: docs/modules/ROOT/pages/getting-started/component_deployment_lifecycle.adoc [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+The lifecycle visualization is currently read-only. However, to customize the build pipelines that {ProductName} executes during this lifecycle, start by using the *Actions* drop-down menu to select *Customize build pipelines*. For detailed instructions on what to do next, visit the xref:getting-started/build_service.adoc[Build pipeline customization] page.
+
+== Logs
```
Comment (dphilips-RH) [inline docs/modules/ROOT/pages/getting-started/component_deployment_lifecycle.adoc:49]: Screenshot here as well?

File: docs/modules/ROOT/pages/getting-started/component_deployment_lifecycle.adoc (hunk 2 above)
Comment (Chr1st1anSears) [reply docs/modules/ROOT/pages/getting-started/component_deployment_lifecycle.adoc:1]: Great point--the way I'm planning on reorganizing the docs will definitely take care of this :+1:

File: docs/modules/ROOT/pages/getting-started/component_deployment_lifecycle.adoc (hunk 1 above)
Comment (Chr1st1anSears) [reply docs/modules/ROOT/pages/getting-started/component_deployment_lifecycle.adoc:11]: I'm moving the table to the end of the doc :)

File: docs/modules/ROOT/pages/getting-started/component_deployment_lifecycle.adoc (hunk 4 above)
Comment (Chr1st1anSears) [reply docs/modules/ROOT/pages/getting-started/component_deployment_lifecycle.adoc]: Created an issue to remind us to talk about this. I'd def like to add screenshots once we have a policy worked out for them!

**Review by dphilips-RH — APPROVED** [review]

**Review by dphilips-RH — APPROVED** [review]
LGTM

**Review by chloeredhat — APPROVED** [review]

**Pull request merged**